rust/src/index/vector/graph/persisted.rs

```diff
@@ -415,7 +415,7 @@ mod tests {
         .unwrap()];
 
         let write_params = WriteParams {
-            max_rows_per_file: 40,
+            max_rows_per_file: 200,
```

Comment:
is it relevant change ?

Reply:
Previously, the dataset was written in one file. This change made it write to multiple files, and the logic this file doesn't seem to handle that properly. I wasn't able to immediately identify what's wrong.